src/compiler/Compiler.js

```diff
@@ -10,9 +10,11 @@
  * governing permissions and limitations under the License.
  */
 
-const fs = require('fs');
+// built-in modules
 const path = require('path');
-
+// declared dependencies
```

Comment:
?? why this comment ?

Reply:
it's a personal preference. i usually group the require stmts by builtin, declared and local dependencies. it helps me reading and understanding the code. 

does this comment bother you?